WcaOnRails/app/controllers/wdc_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class WdcController < ApplicationController
+  def root
```

Comment:
Oh yeah, it's definitely not RESTy! I could rename this to `index`, but maybe that would be more confusing, because it would make the reader think that the controller *is* resty?

Reply:
Alright, actually from this perspective it sounds alright to have it not RESTy ^^